pkg/operator/starter.go

```diff
@@ -30,31 +30,31 @@ func RunOperator(clientConfig *rest.Config, stopCh <-chan struct{}) error {
 		panic(err)
 	}
 
-	operatorConfigInformers := operatorclientinformers.NewSharedInformerFactory(operatorConfigClient, 10*time.Minute)
-	kubeInformersNamespaced := informers.NewFilteredSharedInformerFactory(kubeClient, 10*time.Minute, targetNamespaceName, nil)
+	informerFactory := operatorclientinformers.NewSharedInformerFactory(operatorConfigClient, 10*time.Minute)
+	openshiftKubeApiServerNSInformerFactory := informers.NewSharedInformerFactoryWithOptions(kubeClient, 10*time.Minute, informers.WithNamespace(targetNamespaceName))
```

Comment:
prefixing with kubeInformers helps me narrow down "which informers" before digging into which namespace.  Restore the ordering in the name